### ISSUE TITLE
GSTREAMER BAD 1.18.3 mingw-w64-x86_64-gst-plugins-bad-1.18.3-2-any.pkg.tar.zst

### ISSUE THREAD

**Issue by SonnyWalkman** [msys2/MINGW-packages#8040]
Is there way to open and select build options meson-options for a package?
I require ALL the elements in the bad package for. 1.18.3.
notably I like the  dtmfdetect element which in wrapped up with /ext/spandsp/ By default, these don't seem to be enabled and hence, not built by ninja.
I'm after the libgstspandsp.dll. library.

Is there anyway to select the build options without Pacman or does anyone have the dll available would be a simple fix.

Sonny.

**Comment by Biswa96** [comment]
Just asking. What is the general use case of that plugin?

**Comment by Biswa96** [comment]
Just asking. What is the general use case of that plugin?

**Comment by Biswa96** [comment]
Hi @SonnyWalkman I have added a pull request to enable the plugin. Would you like to test the package from linked pull request? You can download the package by clicking to the PR -> Checks -> Artifacts -> Download the zip file-> Extract the .zst file -> Install with `pacman -U` command.

**Comment by SonnyWalkman** [comment]
Thanks @Biswa96! So quick with a response and fix. Outstanding.
I'll try it once I'm back at my development PC. 
Seems some of the elements and plugins are disabled as default and its not easy to fiddle with build once packaged.
I'd suggest in creating a dev package packages. e.g.  mingw-w64-x86_64-gst-plugins-bad-dev which has everything enabled.
Similarity create dev packages for the common base, good, ugly and libav we all love to like.

Sonny.

**Comment by Biswa96** [comment]
It's a little bit tedious to check all plugins. Many plugins are dependent on other libraries and not all libraries are developed with Win32 support. Also not all plugins are not required in normal day-to-day tasks.

**Comment by SonnyWalkman** [comment]
@Biswa96
I understand however, I'm new to MSYS2 on windows, I'm finding limited packages although I'm great full for the efforts on most of them being available for gstreamer.
On a side note:-
How hard it to package these plugin's from RidgeRun, I use these a lot and it be great to get them in the MSYS2.
Am I able to package these myself?
https://github.com/RidgeRun/gst-interpipe

https://github.com/RidgeRun/gstd-1.x

I have these running on Ubuntu 20.04LTS using the newer meson/ninja builds.

Sonny

**Comment by Biswa96** [comment]
The gst-interpipe can be built here but every test failed. One has to manually test it. The gstd-1.x requires gio-unix which does not seem to be for Win32 platform.

**Comment by SonnyWalkman** [comment]
Hi @Biswa96,
Just an update, I manually installed the 1.18.3 package following your instructions and can confirm dtmfdetect element is listed with gst-inspect-1.0.  I've successfully used the element in my project.
Thanks for trying to package both gstd.1.x and interpipe, disappointing their not able to run under windows msys2. maybe gio-unix  gets win32/64 support in future.

This issue can be closed off.
